README.md

```diff
@@ -1,90 +1,56 @@
-    * Access Token
-    * Access Token Secret
-4. Obtain the SuiteTalk Base URL, which contains the account ID under the company URLs (Setup->Company->Company
-    Information).
-
-        E.g:  https://<ACCOUNT_ID>.suitetalk.api.netsuite.com
-
-
-## Quickstart(s)
-1. Create a Ballerina file and import the following Netsuite module and the others.
-```ballerina
-import ballerinax/netsuite;
-```
-2. Create a NetSuite Client by providing credentials.
-```ballerina
-public function main() returns error? {
-    netsuite:NetsuiteConfiguration nsConfig = {
-        accountId: "<accountId>",
-        consumerId: "<consumerId>",
-        consumerSecret: "<consumerSecret>",
-        token: "<token>",
-        tokenSecret: "<tokenSecret>",
-        baseURL: "<webServiceURL>"
-    };
-    netsuite:Client netsuiteClient = check new(nsConfig);
-```
-3. Use remote functions of the connector to create, read, update, and delete records in NetSuite.
-```ballerina
-    netsuite:RecordInputRef currency = {
-        internalId: "1",
-        'type: "currency"
-    };
-    netsuite:NewAccount account = {
-        acctNumber: "1234567",
-        acctName: "Ballerina NetSuite Demo Account",
-        currency: currency
-    };
-    netsuite:RecordAddResponse|error newAccount = netsuiteClient->addNewAccount(account);
-    if (newAccount is netsuite:RecordAddResponse) {
-        log:printInfo(output.toString());
-    } else {
-        log:printError(newAccount.message());
-    }
-}
-```
-4. Open an terminal and change directory to where your file is saved and use `bal run <YOUR_BALLERINA_FILE_NAME>.bal` command to run the ballerina file.
-
-## Samples
 
-Instantiate the connector by giving authentication details in the HTTP client config, which has built-in support for 
-TBA . NetSuite uses TBA to authenticate and authorize requests. The NetSuite connector can be instantiated 
-in the HTTP client config using the client ID, client secret, access token and access token secret. You may use Ballerina [Configuration variable](https://ballerina.io/learn/by-example/configurable.html) to store your credentials.
 
-**More samples are available [here](samples).**
+Oracle NetSuite is used for Enterprise Resource Planning (ERP) and to manage inventory, track their financial, host e-commerce stores, and maintain Customer Relationship Management (CRM) systems.This module allows you to access the NetSuite's SuiteTalk SOAP Web services API though Ballerina. 
+
+The NetSuite connector can execute CRUD (create, read, update, delete) and search operations to perform business processing on NetSuite records and to navigate dynamically between records.
+ 
+For more information, go to the module(s).
```

Comment:
Suggestion: "For more information, see  [Ballerina Documentation - ballerinax/netsuite](https://central.ballerina.io/ballerinax/netsuite/latest).